configs/sites/tier1/gaea-c5/packages.yaml

```diff
@@ -1,6 +1,6 @@
 packages:
   all:
-    compiler:: [intel@2023.1.0]
+    compiler:: [intel@2023.1.0] # todo: add gcc here
```

Comment:
Just want to double check. Is the intention to wait until after the 1.8.0 release to make these changes for Gaea?

Reply:
Sure, that makes sense. Happy to approve!